pkg/web/router/router.go

```diff
@@ -68,7 +68,7 @@ func SetAppRoutes(r *echo.Group, appsHandler apps.HTTPHandler) {
 	//     description: Invalid id supplied
 	//   404:
 	//     description: App not found
-	r.GET("/apps/:id", appsHandler.GetAppByID) // TODO: Implement correctly the call of the method passing the parameters
+	r.GET("/apps/:id", appsHandler.GetActiveAppByID) // TODO: Implement correctly the call of the method passing the parameters
```

Comment:
Also, could you remove the TODO comments? This endpoint is already done and then it should no longer be here.e

Reply:
:+1:

Reply:
lol removing the comment reduced the code coverage